### PR TITLE
cell_barcode fields are not required for sciATACseq

### PR DESCRIPTION
**Pull request by cebriggs7135**
cell_barcode fields are not required for sciATACseq or SNAREseq either (as was already noted in parentheses)
Added required: FALSE for cell_barcode read, index & size

### CODE REVIEW

File: src/ingest_validation_tools/table-schemas/assays/scatacseq-v1.yaml [hunk 1]
```diff
@@ -71,13 +71,16 @@ fields:
   constraints:
     pattern: '[ATCG]+(\+[ATCG]+)?'
 - name: cell_barcode_read
-  description: Which read file contains the cell barcode.
+  required: FALSE
```
Comment (mccalluc) [inline src/ingest_validation_tools/table-schemas/assays/scatacseq-v1.yaml:74]:
```suggestion
  constraints:
    required: False
```

**Review by mccalluc — CHANGES_REQUESTED** [review]
`required` must be indented under `constraints`. Please correct all three.

File: src/ingest_validation_tools/table-schemas/assays/scatacseq-v1.yaml [hunk 2]
```diff
@@ -71,13 +71,17 @@ fields:
   constraints:
     pattern: '[ATCG]+(\+[ATCG]+)?'
 - name: cell_barcode_read
-  description: Which read file contains the cell barcode.
+  constraints:
+    required: False
+  description: Which read file contains the cell barcode. This field is not required for barcoding by single-cell combinatorial indexing.
 - name: cell_barcode_offset
-  description: Positions in the read at which the cell barcodes start. Cell barcodes are, for example, 3 x 8 bp sequences that are spaced by constant sequences (the offsets). First barcode at position 0, then 38, then 76. (Does not apply to SNARE-seq and BulkATAC.)
+  required: FALSE
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/assays/scatacseq-v1.yaml:78]:
```suggestion
  constraints:
    required: FALSE
```

File: src/ingest_validation_tools/table-schemas/assays/scatacseq-v1.yaml [hunk 3]
```diff
@@ -71,13 +71,17 @@ fields:
...
+  description: Positions in the read at which the cell barcodes start. Cell barcodes are, for example, 3 x 8 bp sequences that are spaced by constant sequences (the offsets). First barcode at position 0, then 38, then 76. (Does not apply to sciATACseq, SNARE-seq and BulkATAC.)
   constraints:
     pattern: '\d+(,\d+)*'
 - name: cell_barcode_size
-  description: Length of the cell barcode in base pairs. Cell barcodes are, for example, 3 x 8 bp sequences that are spaced by constant sequences, the offsets. (Does not apply to SNARE-seq and BulkATAC.)
+  required: FALSE
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/assays/scatacseq-v1.yaml:83]:
```suggestion
  constraints
    required: FALSE
```

**Review by cebriggs7135 — COMMENTED** [review]
Made the requested edits putting ```constraints:``` above ```required: FALSE```. This is ready for merging.

**Comment by mccalluc** [comment]
Failed on 
```
> antibodies TSV errors:
> - 'On row 2, column "uniprot_accession_number", value "G9N9I7" fails because URL returned
>   HTTPSConnectionPool(host=''www.uniprot.org'', port=443): Max retries exceeded with
>   url: /uniprot/G9N9I7 (Caused by NewConnectionError(''<urllib3.connection.HTTPSConnection
>   object at 0x7f720a7a3dd8>: Failed to establish a new connection: [Errno -2] Name
>   or service not known'',)): "https://www.uniprot.org/uniprot/G9N9I7"'
```
Will try re-running... May be sporadic network problem.

**Pull request merged**